src/ui/common/src/components/pages/workflow/id/index.tsx

```diff
@@ -68,7 +73,10 @@ const WorkflowPage: React.FC<WorkflowPageProps> = ({
   const urlSearchParams = parse(window.location.search);
   const location = useLocation();
   const path = location.pathname;
-  const [currentTab, setCurrentTab] = React.useState<string>('Details');
+
+  const [currentTab, setCurrentTab] = useState<string>('Details');
+  const [showTriggerDialog, setShowTriggerDialog] = useState(false);
```

Comment:
I don't know if we are using `trigger` elsewhere in the UI, but it may be easier to follow if we start naming everything by `run workflow` in frontend codes